supplierProduct/tests.py

```diff
@@ -0,0 +1,60 @@
+import pytest
+from supplierProduct.models import supplierProduct, Product, Supplier
+
+
+class TestSupProduct:
+
+    @pytest.fixture
+    def product(self):
+        prod0 = Product(qr_code="AQMLS5C369VMSL7FOVNX5VLSX0ASSD22", product_name="APPLE", description="sweet!!")
+        prod0.save_product()
+        return prod0
+
+    @pytest.fixture
+    def supplier(self):
+        sup0 = Supplier(
+            user_name="ali22",
+            first_name="ali",
+            last_name="ab",
+            password="2345678",
+            user_type="supplier")
+        sup0.save()
+        return sup0
+
+    @pytest.fixture
+    def supplier_product(self, product, supplier):
+        supprod0 = supplierProduct(
+            suppler_product_id=123,
+            user_name=supplier,
+            qr_code=product,
+            price=22, quantity=4)
+        supprod0.save()
+        return supprod0
+
+    @pytest.mark.django_db
+    def test_object_saved_in_db(self, supplier_product):
+        assert supplier_product in supplierProduct.objects.all()
+
+    @pytest.mark.django_db()
+    def test_creation(self, supplier_product):
```

Comment:
As I see it, the only thing that this test is adding is maintenance burden. It's testing that a python object can be created. No need to verify that python works -- Just our code.

Moreover, suppose that tomorrow you change the fixture, you'd also have to change this test.

All in all, it does not monitor regressions on our code (which is the reason to have unit tests), but it does add additional tasks for our code maintenance.